index.js

```diff
@@ -211,16 +189,16 @@ class Logger {
       message = tags;
       tags = [];
     }
-    message = this.serialize(tags, message, this.config);
     if (this.config.defaultTags.length !== 0) {
       tags = this.config.defaultTags.concat(tags);
     }
+    message = typeof message === 'object' ? this.serialize(tags, message, this.config) : message;
```

Comment:
what does this do?

Reply:
serialize-object expects an object and will error if you pass a string, I feel like we should push this check into serialize-object.

Reply:
yeah, ideally we shouldn't be changing any other logic other than swapping the function with the library

Reply:
lets try to make it so we don't really have to change anything else and just swap the function with the lib